lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -278,6 +281,7 @@ spec = do
             jsonRoundtripAndGolden $ Proxy @ApiBlockReference
             jsonRoundtripAndGolden $ Proxy @ApiNetworkInformation
             jsonRoundtripAndGolden $ Proxy @ApiNetworkParameters
+            jsonRoundtripAndGolden $ Proxy @ApiWalletDelegation
```

Comment:
Why not add the two sub-types here for good measure too?… Or maybe unnecessary 🤷‍♂

Reply:
yeah, it is worth adding them

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1318/commits/d4aca5887172e02285be2e9582cbae5cecc15887